sdk/cosmos/azure-cosmos-benchmark/clientHostAndKey.txt

```diff
@@ -0,0 +1,2 @@
+<host1>,<key1>
```

Comment:
could you please change format to the pattern that portal provides for connection string:

```
AccountEndpoint=https://xyz.documents.azure.com:443/;AccountKey=abc;
```
This makes it easier to construct this file copy/pasting from portal

Reply:
Good point 👍  Done